sample/SampleApp/SampleApp.cpp

```diff
@@ -0,0 +1,82 @@
+// ConsoleApplication1.cpp : Defines the entry point for the console application.
+//
+
+#include <windows.h>
+#include <string>
+#include <playfab/PlayFabSettings.h>
+#include <playfab/PlayFabApiSettings.h>
+#include "playfab_c/PlayFabClientApi_c.h"
+
+PlayFabUserHandle userHandle{ nullptr };
+std::string playFabId;
+
+int main()
+{
+    HRESULT hr = PlayFabInitialize("E18D7");
+    assert(SUCCEEDED(hr));
+
+    {
+        PlayFabLoginWithCustomIDRequest request{};
+        request.createAccount = false;
+        request.customId = "GettingStartedGuide";
+
+        XAsyncBlock async{};
+        async.callback = [](XAsyncBlock* async)
+        {
+            PlayFabResultHandle resultHandle{ nullptr };
+            HRESULT hr = PlayFabGetResultHandle(async, &resultHandle);
+            if (SUCCEEDED(hr))
+            {
+                PlayFabLoginWithCustomIDResultGetUserHandle(resultHandle, &userHandle);
+                const char* playFabIdPtr;
+                PlayFabLoginWithCustomIdResultGetPlayFabId(resultHandle, &playFabIdPtr);
+                playFabId = playFabIdPtr;
+            }
+            PlayFabResultCloseHandle(resultHandle);
+        };
+
+        HRESULT hr = PlayFabLoginWithCustomIDAsync(&request, &async);
+        assert(SUCCEEDED(hr));
+
+        hr = XAsyncGetStatus(&async, true);
+        assert(SUCCEEDED(hr));
+    }
+
+    {
+        XAsyncBlock async{};
+        async.callback = [](XAsyncBlock* async)
+        {
+            PlayFabResultHandle handle{ nullptr };
+            HRESULT hr = PlayFabGetResultHandle(async, &handle);
+            if (SUCCEEDED(hr))
+            {
+                PlayFabPlayerProfileModel* profile{ nullptr };

```

Comment:
>PlayFabPlayerProfileModel [](start = 16, length = 25)

the lifetime of the profile returned here is tied to the result handle.  When the result handle is closed, this pointer is invalidated